mpxe/sims/adt7476a.py

```diff
@@ -1,9 +1,12 @@
 from mpxe.protogen import stores_pb2
-from mpxe.protogen import adt7476a_pb2
 
 from mpxe.sims import sim
 
 class Adt7476a(sim.Sim):
+    def __init__(self):
+        self.speed = None
+        self.status = None
```

Comment:
IMO, since these properties are set to/used as arrays of 3 ints elsewhere, they should be initialized as `[0, 0, 0]` rather than None.